docs/ops/convolution/DeformableConvolution_8.md

```diff
@@ -0,0 +1,169 @@
+  * **Range of values**: integer values starting from `0`
+  * **Type**: `int[]`
+  * **Default value**: None
+  * **Required**: *yes*
+  * **Note**: the attribute is ignored when *auto_pad* attribute is specified.
+
+* *dilations*
+
+  * **Description**: *dilations* denotes the distance in width and height between elements (weights) in the filter. For example, *dilation* equal `1,1` means that all the elements in the filter are neighbors, so it is the same as for the usual convolution. *dilation* equal `2,2` means that all the elements in the filter are matched not to adjacent elements in the input matrix, but to those that are adjacent with distance 1.
+  * **Range of values**: integer value starting from `0`
+  * **Type**: `int[]`
+  * **Default value**: None
+  * **Required**: *yes*
+
+* *auto_pad*
+
+  * **Description**: *auto_pad* how the padding is calculated. Possible values:
+    * *explicit* - use explicit padding values from *pads_begin* and *pads_end*.
+    * *same_upper* - the input is padded to match the output size. In case of odd padding value an extra padding is added at the end.
+    * *same_lower* - the input is padded to match the output size. In case of odd padding value an extra padding is added at the beginning.
+    * *valid* - do not use padding.
+  * **Type**: `string`
+  * **Default value**: explicit
+  * **Required**: *no*
+  * **Note**: *pads_begin* and *pads_end* attributes are ignored when *auto_pad* is specified.
+
+
+* *group*
+
+  * **Description**: *group* is the number of groups which *output* and *input* should be split into. For example, *group* equal to 1 means that all filters are applied to the whole input (usual convolution), *group* equal to 2 means that both *input* and *output* channels are separated into two groups and the *i-th output* group is connected to the *i-th input* group channel. *group* equal to a number of output feature maps implies depth-wise separable convolution.
+  * **Range of values**: integer value starting from `1`
+  * **Type**: `int`
+  * **Default value**: `1`
+  * **Required**: *no*
+
+* *deformable_group*
+
+  * **Description**: *deformable_group* is the number of groups in which *offsets* input and *output* should be split into along the channel axis. Apply the deformable convolution using the i-th part of the offsets part on the i-th out.
+  * **Range of values**: integer value starting from `1`
+  * **Type**: `int`
+  * **Default value**: `1`
+  * **Required**: *no*
+
+* *bilinear_interpolation_padding*
+
+  * **Description**: *bilinear_interpolation_padding* is the number of pixels outside of the feature map boundary to apply bilinear interpolation.
+  * **Range of values**: [0, 1]
+  * **Type**: `int`
+  * **Default value**: `0`
+  * **Required**: *no*
+  
+**Inputs**:
+
+*   **1**: Input tensor of type *T* and rank 4. Layout is `NCYX` (number of batches, number of channels, spatial axes Y and X). Required.
```

Comment:
Wrap "Required" and "Optional" with double "*" characters.

Reply:
done